spec/lib/random_assign_utils_spec.rb

```diff
@@ -78,6 +78,34 @@
       end
     end
 
+    context "no users of group can be assigned because are not members of assign_allowed_on_groups groups" do
+      fab!(:topic_1) { Fabricate(:topic) }
+      fab!(:group_1) { Fabricate(:group) }
+      fab!(:user_1) { Fabricate(:user) }
+
+      before do
+        group_1.add(user_1)
+      end
+
+      it "creates post on the topic" do
+        described_class.automation_script!(
+          {},
+          {
+            "assignees_group" => {
+              "value" => group_1.id,
+            },
+            "assigned_topic" => {
+              "value" => topic_1.id,
+            },
+          },
+          automation,
+        )
+        expect(topic_1.posts.first.raw).to match(
+          "Attempted randomly assign a member of `@#{group_1.name}`, but no one was available.",
```

Comment:
Will this break if we ever change the string?

Reply:
I fixed this test and all the other ones.